src/main/content/_includes/head.html

```diff
@@ -47,6 +47,11 @@
   <meta name="twitter:title" content="{% if page.seo-title %}{{ page.seo-title }}{% else %}{{site.title}}{% endif %}" />
   <meta name="twitter:description" content="{{ description }}" />
   <meta name="twitter:image" content="https://openliberty.io/img/twitter_card.jpg" />
+
+  <meta property='og:title' content="{% if page.seo-title %}{{ page.seo-title }}{% else %}{{site.title}}{% endif %}" />
+  <meta property='og:image' content="{% if page.open-graph-image %} {{page.open-graph-image}} {% else %}https://openliberty.io/img/twitter_card.jpg{% endif %}" />
+  <meta property='og:description' content="{{ description }}" />
+  <meta property='og:url' content="{{ page.url | replace:'index.html','' | absolute_url }}" />
```

Comment:
The opengraph website says 'type' is a required property so we might need to add that as well.

Reply:
yes, not sure how I dropped that from the commit, will add it back along with the update to head-meta for docs